main.py

```diff
@@ -32,9 +32,6 @@ def pagecountbynamespace( self ):
 	def pages_with_most_revisions( self ):
 		self.rep.pages_with_most_revisions()
 
-	def blank_pages( self ):
```

Comment:
Not sure why this has been dropped. Is the report no longer needed? Sorry I haven't looked at this stuff in a while.